### ISSUE TITLE
Figuring out the output layer of our neural network

### ISSUE THREAD

**Issue by wa1234** [KisBak/DuckieTown_OD_Movidius#1]
- [ ] Figuring out the used activation function and the number of output parameters.
- [ ] Figuring out the feed for the loss function.
- [ ] Figuring out the loss function
- [ ] Create a section in the main documentation file and document what you have done
- [ ] Write the python code of the output layer in the proper python file

Deadline: oct. 27
Estimated remaining time: 8h
Reviewer: @wa1234 
Depends on:

**Comment by wa1234** [comment]
Történt valami előrelépés ezügyben? Kell valakinek segítség, stb.? Módosítsuk a határidőt? @KisBak , @fellowess ?

**Comment by wa1234** [comment]
A yolov3 tartalmazza, így nincs értelme
